### DISCUSSION TITLE
Beercss Snippets and Codepen

### DISCUSSION THREAD

**Discussion by KicKerBNU** [beercss/beercss#125] [Show and tell]
Hello community, im working on updates of the beercss snippet, its available on VS Code Extension for download.
Link for download: [beercss snippet](https://marketplace.visualstudio.com/items?itemName=evertonbuzzi.beercss-snippet).

Also working on updates of the beercss codepen, its availbale on [beercss codepen](https://codepen.io/collection/XydYMB?grid_type=list)

All suggestions are welcome.
Cheers 🍺

**Comment by KicKerBNU** [comment]
Snippets updated based on the release 3.2.10

**Comment by laurentpayot** [comment]
Cool :+1:  I’m using [Elm](https://guide.elm-lang.org/) but I guess it’s a bit too niche :wink:

**Comment by KicKerBNU** [comment]
Codepen updated based on the release 3.11.17
Snippets updated based on the release 3.3.0
